### DISCUSSION TITLE
How to customise colours for dark theme?

### DISCUSSION THREAD

**Discussion by HubKing** [ant-design-blazor/ant-design-blazor#2929] [Q&A]
As far as I know, Blazor does not have light/dark mode support, so I had just modified /wwwroot/css/site.css to make the text white and the background black. 

After applying AntBlazor, the text became black and the background became white. It seems that `ant-design-blazor.css` is overriding `site.css`, but I do not know where `ant-design-blazor.css` is. I have searched the Discussions list and there were 2 questions about colours, but the linked documentation seemed to be React or something, which I don't know about, not Blazor. 

So, is it difficult to change colours for Blazor? Basically, I want to use a dark theme.

**Comment by ElderJames** [comment]
Please try change `ant-design-blazor.css` to  `ant-design-blazor.dark.css`.
